ref/general/mp-openapi.adoc

```diff
@@ -10,47 +10,80 @@
+
+Developers can communicate with the REST API in their web service only if they have the relevant information to do so. It is important to document APIs when they are published so that other developers can use them.
+
+There are two main ways to generate a link:https://swagger.io/docs/specification/about/[OpenAPI] document. The developer can generate documentation of the REST API from the source code of the REST API itself. The reference document thus produced, lists all the API endpoints and descriptions of how to use them. Alternatively, the developer can start by designing the API in the form of structured documentation that they can then augment with annotations in the code. The developer can even generate the stubs for the source code, including client libraries for the API, from this manually created documentation.
+
+== Processing annotations
+
+You can augment the existing code with OpenAPI annotations, which provide detailed documentation for the REST API.
+
+For example, the following code example shows JAX-RS code to request the list of host names in the inventory. The code has been annotated with OpenAPI annotations to provide more useful and explanatory information about the operation:
+
+[source,java]
+----
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @APIResponse(
+        responseCode = "200",
+        description = "host:properties pairs stored in the inventory.",
+        content = @Content(
+            mediaType = "application/json",
+            schema = @Schema(
+                type = SchemaType.OBJECT,
+                implementation = InventoryList.class)))
+    @Operation(
+        summary = "List inventory contents.",
+        description = "Returns the currently stored host:properties pairs in the "
+        + "inventory.")
+    public InventoryList listContents() {
+        return manager.list();
+    }
+----
+
+In the following OpenAPI documentation example that is generated from the example code, the JAX-RS framework generates the path /inventory/systems based on the value of the @Path annotation, and the value of the operationId key (listContents) based on the name of the method (listContents()). The other parts of the document are generated from the OpenAPI annotations with which the developer annotated the code. For example, the @Operation annotation provides more meaningful information about what the operation does and why; in this case, the operation returns a list of the host names with properties of each host that are currently stored by the inventory web service. This is described by the summary and description keys in the @Operation annotation:
```

Comment:
In the phrase "the path /inventory/systems", put the noun after the path name, so change to "the /inventory/systems path". 
Change "OpenAPI annotations with which the developer annotated the code" to "OpenAPI annotations that the developer annotated the code with", this might help to make the wording a bit simpler